huaweicloud/services/lb/resource_huaweicloud_lb_pool.go

```diff
@@ -70,13 +73,18 @@ func ResourcePoolV2() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 				ForceNew: true,
+				Computed: true,
+				AtLeastOneOf: []string{
```

Comment:
conflicts with docs: ```Note: Exactly one of LoadbalancerID or ListenerID must be provided.```

Reply:
docs updated, it's 'at least one of' in the api